how-to-play/index.md

```diff
@@ -110,10 +108,27 @@ You can only recover sanity by using Sanity Pills.
 
 <details>
   <summary>Advanced</summary>
- 
-  - you lose sanity at the rate of `deltatime * ghost_sanity_strength` where all ghosts have a `ghost_sanity_strength` of 0.02 but Shade has 0.4
-  - sanity can never drop below 50% in Setup Phase
-  - sanity drains slower in Setup Phase (0.09 vs 0.12)
+ BEFORE READING - This is technical information! It could spoil your play experience knowing too much about how the game works. This information is datamined and could change with future updates, not be implemented in the game, or not fully understood. Read below at your own discretion.
+  
+  # Sanity Information
+  
+  - When looking at a manifested ghost you lose sanity at the rate of `deltatime * ghost_sanity_strength` where all ghosts have a `ghost_sanity_strength` of 0.02 but Shade has 0.04 (It is unknown how or if this is changed at higher difficulties).
+  - Sanity can never drop below 50% in Setup Phase.
+  - Sanity drains slower in Setup Phase (0.09 vs 0.12).
+  - Sanity meters in the van only update every 2 seconds.
+  - Sanity meters in the van fluctuate from -2 to +3 of your actual sanity. EX: If I have 80 sanity the display could show between 78 and 83 sanity.
+  
+  # Equipment Information
+  
+  - Crucifix has a range of 4 meters for all ghosts except the Banshee where the crucifix will have a 6 meter range.
```

Comment:
I've heard this is 3m. Can you confirm it is 4m?

Reply:
The code says 4m in it, i can check again later to see if maybe it was changed.

Reply:
It is 3m in this patch https://discordapp.com/channels/435431947963990026/661365069006897154/760599252480163840